packages/features/tokens-provider/src/TokensContextProvider.tsx

```diff
@@ -164,6 +167,9 @@ export const TokensContextProvider: React.FC = ({ children }) => {
       pTokens[symbol].balance = balance;
       return { ...pTokens };
     });
+    let today: any = new Date();
+    let today2 = timeConverter(BigInt(today * 1000000))
```

Comment:
let today: any = new Date();
 let today2 = timeConverter(BigInt(today * 1000000))`

make this to something like this:
today =  timeConverter(BigInt(new Date().getTime() * 1000000))

and don't use let, its all const